lib/fog/parsers/terremark/instantiate_vapp_template.rb

```diff
@@ -4,18 +4,18 @@ module Terremark
       class InstantiateVappTemplate < Base
         def reset
           @property_key
-          @response = { 'Links' => [] }
+          @response = { "Links" => [] }
         end
 
         def start_element(name, attributes)
           super
           case name
-            when 'Link'
-              link = extract_attributes(attributes)
-              @response['Links'] << link
-            when 'VApp'
-              vapp_template = extract_attributes(attributes)
-              @response.merge!(vapp_template.reject {|key, value| !['href', 'name', 'size', 'status', 'type'].include?(key)})
+          when "Link"
+            link = extract_attributes(attributes)
+            @response["Links"] << link
+          when "VApp"
+            vapp_template = extract_attributes(attributes)
+            @response.merge!(vapp_template.reject {|key, value| !["href", "name", "size", "status", "type"].include?(key)})
```

Comment:
Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.<br>Space between { and | missing.<br>Space missing inside }.